tests/Unit/Domain/Test_StrahlkorperTransformations.cpp

```diff
@@ -92,13 +93,18 @@ void test_strahlkorper_in_different_frame() {
             2.0, ylm.theta_phi_points(), 1.0,
             std::array<double, 3>{{0.1, 0.2, 0.3}}));
     strahlkorper_expected.reset(new ylm::Strahlkorper<DestFrame>(
-        l_max, l_max, new_radius, strahlkorper_grid_center));
+        l_max, l_max, new_radius, strahlkorper_src_center));
+  } else if constexpr (std::is_same_v<SrcFrame, ::Frame::Inertial> and
+                       std::is_same_v<DestFrame, ::Frame::Distorted>) {
+    strahlkorper_expected.reset(new ylm::Strahlkorper<DestFrame>(
```

Comment:
Clang tidy fails on this line, but I don't understand what it's mad about

Reply:
Discussion on 1/25/24 call: we agree to disable the check that this trips. If I need to make changes before merging this PR (I'm guessing that will be the case), I can disable that check as part of this PR once someone tells me how

Reply:
Yeah an extra commit sounds good.